server/rest/src/main/java/org/infinispan/rest/stream/CacheEntryStreamProcessor.java

```diff
@@ -112,28 +116,51 @@ private static byte[] metadataBytes(CacheEntry<?, ?> currentEntry) {
       if (!(currentEntry instanceof InternalCacheEntry)) {
          return null;
       }
+
+      Metadata metadata = currentEntry.getMetadata();
       InternalCacheEntry ice = (InternalCacheEntry) currentEntry;
       // add metadata
-      long lifespanInSeconds = ice.getLifespan();
+      long lifespanInSeconds = metadata.lifespan();
```

Comment:
why change this line and the above?

Reply:
I thought it was better to use metadata instead. I'm reverting those changes.